store/src/main/java/com/dropbox/android/external/store4/StoreBuilder.kt

```diff
@@ -138,13 +138,13 @@ interface StoreBuilder<Key : Any, Output : Any> {
 
 @FlowPreview
 @ExperimentalCoroutinesApi
-private class BuilderImpl<Key : Any, Output : Any>(
+ class BuilderImpl<Key : Any, Output : Any>(
```

Comment:
There should be no need to expose new private types to make this change. Unless I missed something, all you're doing is convert the input params from Rx to Flow

Reply:
was a in progress change will revert

Reply:
aha I can use the old factories, nice catch